src/Compilers/Core/CommandLine/CompilerServerLogger.cs

```diff
@@ -113,11 +117,18 @@ public static void Log(string message)
                 string output = prefix + message + "\r\n";
                 byte[] bytes = Encoding.UTF8.GetBytes(output);
 
+                // PROTOTYPE(NullableDogfood): There should be no warning on de-referencing s_loggingStream since checked above
```

Comment:
Agreed, there should be no warning dereferencing the field. We should track static fields and properties. Please log a bug.